### ISSUE TITLE
zoxide can't access it's datase (when exportet to server)

### ISSUE THREAD

**Issue by MartyMcFlyInTheSky** [ajeetdsouza/zoxide#864]
I'm using zoxide on my local laptop running ubuntu where it works like a charm. However, since I'm hopping to our production machines frequently I want to make use of zoxide there, so I'm cloning my dotfiles containing the zoxide binary to the servers in `/tmp/dotfiles`. Basically this becomes my new XDG_HOME. The zoxide binary lives in `/tmp/dotfiles/.local/bin/` and since zoxide respects the XDG norm it correctly creates a database file in $XDG_DATA_HOME/zoxide which expands to `/tmp/dotfiles/.local/share/zoxide` . Everything looks fine so far.

However, I noticed that zoxide fails to read anything from the database. Using `zi` to access the list will always yield
`zoxide: no match found`
even though I should be inserting entries with `z`...

I checked permissions on the db file and they seem fine:
`-rw-r--r-- 1 process process 12 Aug 2 09:56 db.zo`

So there's read and write permission for the process user which is the one I'm using. Still, one oddity is that on my local machine (Ubuntu), the permissions are:

`-rw-rw-r-- 1 myuser myuser 2310 Aug 2 11:43 db.zo`

So group has write permissions as well. Idk how important this is, I think write from user should do it?

In summary, zoxide has the right perception of where the file is (it creates it at the right spot at least). But then it doesn't retrieve the data correctly or save it. Again, locally everything works as expected.

We're running RHEL 9.2 on these servers.

**Comment by MartyMcFlyInTheSky** [comment]
Update: It now seems to work in some conditions. I haven't changed a thing. Still kinda confused why this appeared. I will test for another 1-2 weeks and then update the status of this ticket.

**Comment by Bongoros** [comment]
Try moving the db.zo file to another directory, database file can be used in other application.
I had the same issue and when i closed the app that was conflicting everything worked fine.

**Comment by MartyMcFlyInTheSky** [comment]
@Bongoros Interesting find, thanks for reporting. That could well be the case since now I didn't run into any further problems. I'll close this ticket (as I promised I just now see).
